libraries/Microsoft.Bot.Builder/FileTranscriptLogger.cs

```diff
@@ -49,8 +49,8 @@ public FileTranscriptLogger(string folder = null, bool unitTestMode = true)
                 Directory.CreateDirectory(folder);
             }
 
-            this.folder = folder;
-            this.unitTestMode = unitTestMode;
+            this._folder = folder;
```

Comment:
>_folder [](start = 17, length = 7)

This does NOT meet the dotnet framework style guidlines [https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members)